windows-pull-run.bat

```diff
@@ -110,14 +112,22 @@ IF NOT "%1"=="" (
 @REM If a tag was not specified, use "latest" as the tag
 IF "%tag%"=="" SET tag=latest
 
+@REM Setting timezone correctly
+
+SET timezone2=`powershell -NoExit -EncodedCommand "JAB0AHoAcwB0AHIAaQBuAGcAIAA9ACAAJwBbAHsAIgBXAGkAbgBkAG8AdwBzACIAOgAiAEQAYQB0AGUAbABpAG4AZQAgAFMAdABhAG4AZABhAHIAZAAgAFQAaQBtAGUA IgAsACIATABpAG4AdQB4ACIAOgAiAEUAdABjAC8ARwBNAFQAKwAxADIAIgB9ACwAewAiAFcAaQBuAGQAbwB3AHMAIgA6ACIAVQBUAEMALQAxADEAIgAsACIATABpAG4A dQB4ACIAOgAiAEUAdABjAC8ARwBNAFQAKwAxADEAIgB9ACwAewAiAFcAaQBuAGQAbwB3AHMAIgA6ACIAQQBsAGUAdQB0AGkAYQBuACAAUwB0AGEAbgBkAGEAcgBkACAA VABpAG0AZQAiACwAIgBMAGkAbgB1AHgAIgA6ACIAQQBtAGUAcgBpAGMAYQAvAEEAZABhAGsAIgB9ACwAewAiAFcAaQBuAGQAbwB3AHMAIgA6ACIASABhAHcAYQBpAGkA YQBuACAAUwB0AGEAbgBkAGEAcgBkACAAVABpAG0AZQAiACwAIgBMAGkAbgB1AHgAIgA6ACIAUABhAGMAaQBmAGkAYwAvAEgAbwBuAG8AbAB1AGwAdQAiAH0ALAB7ACIA VwBpAG4AZABvAHcAcwAiADoAIgBNAGEAcgBxAHUAZQBzAGEAcwAgAFMAdABhAG4AZABhAHIAZAAgAFQAaQBtAGUAIgAsACIATABpAG4AdQB4ACIAOgAiAFAAYQBjAGkA ZgBpAGMALwBNAGEAcgBxAHUAZQBzAGEAcwAiAH0ALAB7ACIAVwBpAG4AZABvAHcAcwAiADoAIgBBAGwAYQBzAGsAYQBuACAAUwB0AGEAbgBkAGEAcgBkACAAVABpAG0A ZQAiACwAIgBMAGkAbgB1AHgAIgA6ACIAQQBtAGUAcgBpAGMAYQAvAEEAbgBjAGgAbwByAGEAZwBlACIAfQAsAHsAIgBXAGkAbgBkAG8AdwBzACIAOgAiAFUAVABDAC0A MAA5ACIALAAiAEwAaQBuAHUAeAAiADoAIgBFAHQAYwAvAEcATQBUACsAOQAiAH0ALAB7ACIAVwBpAG4AZABvAHcAcwAiADoAIgBQAGEAYwBpAGYAaQBjACAAUwB0AGEA bgBkAGEAcgBkACAAVABpAG0AZQAgACgATQBlAHgAaQBjAG8AKQAiACwAIgBMAGkAbgB1AHgAIgA6ACIAQQBtAGUAcgBpAGMAYQAvAFQAaQBqAHUAYQBuAGEAIgB9ACwA ewAiAFcAaQBuAGQAbwB3AHMAIgA6ACIAVQBUAEMALQAwADgAIgAsACIATABpAG4AdQB4ACIAOgAiAEUAdABjAC8ARwBNA`
```

Comment:
Can you guys please have the PowerShell file separate and spawn that here? I can't really review this line